src/components/InlinePickList/InlinePickList.tsx

```diff
@@ -45,6 +47,8 @@ const InlinePickList: React.FunctionComponent<InlinePickListProps> = (props) =>
         </ReadOnlyField>
     }
 
+    const {t} = useTranslation()
```

Comment:
You should move this on top of the component, or it might crash if there was a `useTranslation` call during first render and if early return was hit due to `props.readonly` being true during the second render, and vice versa (see https://github.com/tesler-platform/tesler-ui/issues/167)